AppCenter/AppCenter/MSAppCenter.h

```diff
@@ -77,6 +77,13 @@
  */
 + (BOOL)isConfigured;
 
+/**
+ * Check whether app is running in App Center Test.
+ *
+ * @return true if running in App Center Test, false otherwise.
+ */
++ (BOOL)isRunningInAppCenterTestCloud;
```

Comment:
The doc says App Center Test but the method name says Test Cloud. Need it to be consistent.